doc/whats_new/v1.0.rst

```diff
@@ -12,6 +12,13 @@ Version 1.0.2
 Changelog
 ---------
 
+:mod:`sklearn.linear_model`
+...........................
+
+- |Fix| Fixed :class:`linear_model.ElasticNetCV` call with l1_ratio=0 producing warning
+  incorrectly.
+  :pr:`21724` by :user:`Yar Khine Phyo <yarkhinephyo>`.
```

Comment:
Updating an unclear warning message feels like it belongs in `1.1`. We normally use 1.0.X releases for numerical or API bugs. May we move the whats new to `doc/whats_new/v1.1.rst`?